packages/subgraph/src/mappings/cfav1.ts

```diff
@@ -219,22 +287,109 @@ export function handleStreamUpdated(event: FlowUpdated): void {
         event,
         oldFlowRate,
         stream.id,
-        newStreamedUntilLastUpdate
+        newStreamedUntilLastUpdate,
+        newDeposit
     );
-    handleStreamPeriodUpdate(flowUpdateEvent, oldFlowRate, stream);
+    handleStreamPeriodUpdate(flowUpdateEvent, oldFlowRate, stream.id);
```

Comment:
StreamPeriods should also be aware of deposits...

Reply:
donzo